src/ts/ui/runner/burninprompt.ts

```diff
@@ -0,0 +1,50 @@
+
+const ACCEPTABLE_RANGE_IN_STDDEV = 2;
+
+const MIN_LENGTH = 10;
+
+export class BurninPrompt {
+
+  evalSeriesStdDev(series:number[]) : number {
+    let index = -1;
+    if (series.length >= MIN_LENGTH) {
+      const midpoint = Math.ceil(series.length / 2);
+      const secondHalf = series.slice(midpoint);
+      /* get the stddev of the second half */
+      const sum = secondHalf.reduce((tot, c)=>tot+(c||0), 0);
+      const avg = sum / secondHalf.length;
+      const sumDeltaSq = secondHalf.reduce((tot, c)=>{
+        const d = c - avg;
+        return tot + d * d;
+      }, 0);
+      const stdDev = Math.sqrt(sumDeltaSq/secondHalf.length);
+      for (let i = midpoint - 1; i >= 0; i--) {
```

Comment:
I'd even start at `series.length - 1` instead of `midpoint - 1`; there's no guarantee that the second half of the series is already past some rough burn-in cutoff

Reply:
Just pushed up a change to start at `last index - 1`, so that we have at least one sample after the knee, but if that's too arbitrary, let me know.